watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -138,7 +140,7 @@ def main(bio_P=True):
     return m, results
 
 
-def build(bio_P=False):
+def build(bio_P=True):
```

Comment:
I'm wondering if there's a reason for changing the default values (False to True). It would be helpful to document this, even with a simple comment.

Reply:
False is what this flowsheet should be using ideally, but it's currently not as stable - hence why I've had to remove testing the `bio_P=False` config option from the flowsheet test file. This is also why `bio_P=True` is set as the default in the GUI.